client/src/components/Menu/index.tsx

```diff
@@ -20,29 +40,81 @@ interface IMenuProps {
     newUsername: string;
 }
 
-const Menu = ({loggedUsername, newUsername}: IMenuProps): JSX.Element => (
-    <div className='container-fluid header-menu'>
-        <div className='row justify-content-between bg-white'>
-            <Link to='/'>
-                <img src={logo}
-                     alt='logo'
-                     width={110}
-                     className='mb-3 sm-mb-2 sm-mr-5 pt-1 logo'/>
-            </Link>
-            <div className='w-30 mt-4 mb-4 form-group search d-md-block d-none'>
-                <span className='fa fa-search form-control-feedback'/>
-                <Input placeholder='Search' type='search' className='form-control px-4' disabled={true}/>
-            </div>
-            <div className='mt-4'>
-                <i className='fa fa-compass fa-lg pt-2 pb-2 pr-4 sm-pt-0 icon text-muted'/>
-                <i className='fa fa-heart-o fa-lg pr-4 icon text-muted'/>
-                <Link to={`/profile/${newUsername ? newUsername : loggedUsername}`}>
-                    <i className='fa fa-user-o fa-lg pr-3 icon'/>
-                </Link>
+class Menu extends React.Component<IMenuProps> {
+    public state: any = {
+        isMenuOpen: false,
+    };
+
+    public toggle = (e: any): void => {
```

Comment:
any?

Reply:
I can't change this any another type, which type i can write other any, can you help me?

Reply:
e: React.ChangeEvent<HTMLInputElement> ??

Reply:
![image](https://user-images.githubusercontent.com/49918923/65387076-1b7de800-dd4c-11e9-8b56-7e5747429dd0.png)

Reply:
![image](https://user-images.githubusercontent.com/49918923/65387081-36e8f300-dd4c-11e9-94a5-9fc1f84235e0.png)

Reply:
fixed